cherrypicks_test.go

```diff
@@ -181,7 +181,7 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 	}
 	defer os.RemoveAll(tmpdir)
 
-	gitSetup := exec.Command("git", "clone", "https://github.com/mendersoftware/integration.git", tmpdir)
+	gitSetup := exec.Command("git", "clone", "https://github.com/mender-test-bot/integration.git", tmpdir)
```

Comment:
I agree with the principle, but not in the actual implementation.

I think we should still clone `mendersoftware` repo, but then just checkout a known git sha. We can for example take today's master, and we will be updating it as we need (when for whatever reason we have a regression that needs a new test or a new feature that requires newer master)

Reply:
But the new release will always mess up the result of running the tests. i.e., they will have to be upgraded with every release

Reply:
Discussed f2f: Okey! :+1: